tab-style navigation (#871)
SAP/luigi: docs/general-settings.md

}
```

> * **hideNavigation** disables Luigi's default out-of-the-box navigation when set to `true`. As a result, neither the left, top nor tab navigation is visible in your application, and you can implement your own navigation UI. By default, the parameter is set to `false`, which means the navigation is enabled.

Reviewer: ```suggestion
* **hideNavigation** disables Luigi's default out-of-the-box navigation when set to `true`. This means that top, side, and tab navigation is no longer visible and you can implement your own navigation UI. By default, the parameter is set to `false`, which means the navigation is enabled.
```
